input/docs/handbook/view-models/index.md

```diff
@@ -1,167 +1,112 @@
 
 ```cs
 this.WhenAnyValue(x => x.Name)
     .Select(x => x.Split(' ')[0])
     .ToProperty(this, x => x.FirstName, out firstName);
 ```
 
-Here, `ToProperty` creates an `ObservableAsPropertyHelper` instance which will
-signal that the "FirstName" property has changed. `ToProperty` is an extension
-method on `IObservable<T>` and semantically acts like a "Subscribe".
+Here, `ToProperty` creates an `ObservableAsPropertyHelper` instance which will signal that the "FirstName" property has changed. `ToProperty` is an extension method on `IObservable<T>` and semantically acts like a "Subscribe".
 
-### Best Practices
+# Best Practices
 
-One of the core concepts of Functional Reactive Programming, is that instead
-of writing *imperative code* (i.e. "Do A right now, then do B, then do C"), we
-want to write *Functional, Declarative code* - instead of writing event
-handlers and methods to change properties, we want **to Describe how properties
-are related to each other**, using `WhenAny` and `ToProperty`.
+One of the core concepts of Functional Reactive Programming, is that instead of writing *imperative code* (i.e. "Do A right now, then do B, then do C"), we want to write *Functional, Declarative code* - instead of writing event handlers and methods to change properties, we want **to Describe how properties are related to each other**, using `WhenAny` and `ToProperty`.
 
-As a result, almost all of the interesting code in a well-written ReactiveUI
-ViewModel will be in the *constructor*; this code will describe how the
-properties in the ViewModel are related to each other. Your goal when writing
-the code for a ViewModel, is to take statements that describe how the view
-*should* work in terms of its commands and properties, and to translate them
-into things to put into the constructor. For example:
+As a result, almost all of the interesting code in a well-written ReactiveUI ViewModel will be in the *constructor*; this code will describe how the properties in the ViewModel are related to each other. Your goal when writing the code for a ViewModel, is to take statements that describe how the view *should* work in terms of its commands and properties, and to translate them into things to put into the constructor. For example:
 
 * "The Login button can be pressed when the username and password aren't blank"
 * "The error message should be cleared 10 seconds after it is displayed"
 * "The DirectMessageToSend object consists of the target user and the message
    to send""
 
-All of these statements are concise descriptions of parts of how your UI
-should work, and these statements can all be directly translated into Rx
-expressions in your ViewModel constructor.
+All of these statements are concise descriptions of parts of how your UI should work, and these statements can all be directly translated into Rx expressions in your ViewModel constructor.
+
+# Dealing with Boilerplate code
```

Comment:
Change "Dealing with Boilerplate" to "Managing Boilerplate", sounds more inviting.